src/plugins/usage_collection/README.md

```diff
@@ -177,7 +177,7 @@ New fields added to the telemetry payload currently mean that telemetry cluster
 
 There are a few ways you can test that your usage collector is working properly.
 
-1. The `/api/stats?extended=true` HTTP API in Kibana (added in 6.4.0) will call the fetch methods of all the registered collectors, and add them to a stats object you can see in a browser or in curl. To test that your usage collector has been registered correctly and that it has the model of data you expected it to have, call that HTTP API manually and you should see a key in the `usage` object of the response named after your usage collector's `type` field. This method tests the Metricbeat scenario described above where `callCluster` wraps `callWithRequest`.
+1. The `/api/stats?extended=true&legacy=true` HTTP API in Kibana (added in 6.4.0) will call the fetch methods of all the registered collectors, and add them to a stats object you can see in a browser or in curl. To test that your usage collector has been registered correctly and that it has the model of data you expected it to have, call that HTTP API manually and you should see a key in the `usage` object of the response named after your usage collector's `type` field. This method tests the Metricbeat scenario described above where `callCluster` wraps `callWithRequest`.
```

Comment:
There are so many different use cases for referring to `legacy` I think it's worth while adding a one-liner about what adding `&legacy=true` does to the field names. That being said, it's easy enough to figure it out when going through the [code for the API](https://github.com/elastic/kibana/blob/master/src/legacy/server/status/routes/api/register_stats.js#L109).

Reply:
I agree! I've created https://github.com/elastic/kibana/issues/75273 to properly document that API 🙂